sql/driver/conn.go

```diff
@@ -178,13 +178,18 @@ func (c *conn) Send(call sqlwire.Call) (*rows, error) {
 			} else if datum.IntVal != nil {
 				t[j] = *datum.IntVal
 			} else if datum.UintVal != nil {
-				t[j] = *datum.UintVal
+				// uint64 not supported by the driver.Value interface.
```

Comment:
Not related to this PR specifically, but shouldn't this code be in an implementation of [Valuer](http://godoc.org/database/sql/driver#Valuer) on `Datum`?